entproto/fieldmap.go

```diff
@@ -79,6 +79,20 @@ func (m FieldMap) Edges() []*FieldMappingDescriptor {
 	sort.Slice(out, func(i, j int) bool {
 		return out[i].PbStructField() < out[j].PbStructField()
 	})
+
+	return out
+}
+
+func (m FieldMap) Enums() []*FieldMappingDescriptor {
+	var out []*FieldMappingDescriptor
+	for _, f := range m {
+		if f.IsEnumFIeld {
+			out = append(out, f)
+		}
+	}
+	sort.Slice(out, func(i, j int) bool {
+		return out[i].PbStructField() < out[j].PbStructField()
+	})
```

Comment:
I saw this sorting a few times here - why it's needed? Why not keep the same order as in the schema?

Reply:
b/c:
```go
// FieldMap contains a mapping between the field's name in the ent schema and a FieldMappingDescriptor.
type FieldMap map[string]*FieldMappingDescriptor`
```
its a map, which doesn't have stable order when ranged